### ISSUE TITLE
What is the loss value of the trained model？

### ISSUE THREAD

**Issue by YiHanZhu-lele** [ClubAI/MonoDepth-PyTorch#38]
I want to know the  loss value of the provided trained model.
the provided trained model's parameter:
model: ‘resnet18_md’
epochs: 200,
learning_rate: 1e-4,
batch_size: 8,
adjust_lr: True,
do_augmentation: True
Thank you.

**Comment by askorikov** [comment]
Unfortunately, this value is too difficult to retrieve at this point. You can try to repeat the training to get it out.
